lib/test_utils.dart

```diff
@@ -16,6 +16,26 @@ CorePalette generateCorePalette(int Function(int index) generator) =>
       List<int>.generate(CorePalette.size * TonalPalette.commonSize, generator),
     );
 
+/// Sample [CorePalette]s to be used to mock dynamic color in tests & development.
```

Comment:
What are these samples needed for if the tests don't use them?

Reply:
these are currently internal-only and there's not a good reason for them not to be available to all developers for their tests